addons/l10n_ar/models/res_partner.py

```diff
@@ -52,7 +52,7 @@ def _compute_l10n_ar_vat(self):
         one is not found """
         recs_ar_vat = self.filtered(lambda x: x.l10n_latam_identification_type_id.l10n_ar_afip_code == '80')
         for rec in recs_ar_vat:
-            rec.l10n_ar_vat = rec.vat
+            rec.l10n_ar_vat = stdnum.ar.cuit.clean(rec.vat, ' -.')
```

Comment:
Whe we are using also dot on the clean? It's possible to have dots on CUIT?
Perhaps we can use [`stdnum.ar.cuit.compact`](https://github.com/arthurdejong/python-stdnum/blob/master/stdnum/ar/cuit.py#L48) instead
What do you think @zaoral ?

Reply:
Hi @jjscarafia
Since I was not sure how the user could separate the CUIT number, I added "." just as an extra check. Yes, I can use compact function() what this function does is the same a clean(vat, ' -') just without the point.

I think we can use the compact function, But I will like to have an extra validation to ensure that the cuit number is actually only numbers. Do you think will be good to run an is_digit() or similar to ensure that the cuit we are cleaning is actually only numbers in this compute field?

Reply:
Hi @zaoral 
I've try to add characters or dots and they are not allowed. I believe that is already catched by the validation [here](https://github.com/odoo/odoo/blob/c9fde2ab7e20e549b39d60573ded7766fa73d9c8/addons/l10n_ar/models/res_partner.py#L104) and later [here](url)

Reply:
so, just using "compact" seems enough to me

Reply:
okis, changed and pushed